src/main/java/gregtech/common/metatileentities/electric/MetaTileEntityRockBreaker.java

```diff
@@ -101,4 +101,9 @@ protected boolean shouldSearchForRecipes() {
             return hasValidFluids && super.shouldSearchForRecipes();
         }
     }
+
+    @Override
+    public boolean getTerrainResistance(){
+        return true;
```

Comment:
This machine should also have the tooltip applied to it if the weather explosion config is true.

Reply:
TieredMTE class has this code so this machine already has the tooltip
```
    @Override
    public void addInformation(ItemStack stack, @Nullable World player, @Nonnull List<String> tooltip, boolean advanced) {
        super.addInformation(stack, player, tooltip, advanced);
        if (ConfigHolder.machines.doTerrainExplosion && getIsWeatherOrTerrainResistant())
            tooltip.add(I18n.format("gregtech.universal.tooltip.terrain_resist"));
    }
```